dcos/namerd.dtab

```diff
@@ -1,6 +1,5 @@
-/srv        => /io.l5d.marathon;
-/www        => /srv/web;
-/host       => /srv | /$/io.buoyant.http.anyHostPfx/www;
-/method     => /$/io.buoyant.http.anyMethodPfx/host;
-/http/1.1   => /method;
-/http/1.0   => /$/io.buoyant.http.anyMethodPfx/www;
+/srv         => /io.l5d.marathon ;
+/http/1.0    => /srv/web ;
+/host/*      => /srv/web ;
+/host        => /srv     ;
+/http/1.1/*  => /host    ;
```

Comment:
recommend holding off on merging this until https://github.com/mesosphere/universe/pull/438 ships. or, create a namerd-0.4.0.dtab version